loglist/logfilter_test.go

```diff
@@ -58,15 +58,14 @@ func TestActiveLogs(t *testing.T) {
 	}
 }
 
-func certChain() []*x509.Certificate {
+func cert() *x509.Certificate {
```

Comment:
It looks like `cert()` and `caCert()` are only used in `TestCompatible()` - you could just move their code to the start of that test and get rid of the functions.

Reply:
Moved into the test.